tsconfig.web.json

```
@@ -1,6 +1,6 @@
 {
     "extends": "./tsconfig.shared.json",
-    "include": ["src/renderer/env.d.ts", "src/renderer/typed-router.d.ts", "src/renderer/**/*", "src/renderer/**/*.vue", "src/preload/*.d.ts"],
+    "include": ["src/main/", "src/renderer/env.d.ts", "src/renderer/typed-router.d.ts", "src/renderer/**/*", "src/renderer/**/*.vue", "src/preload/*.d.ts"],
```

We ideally don't want to include the main folder here. Everything in `main` runs in a Node context, so it should error if we try to use browser-only APIs, like `window`. It's covered by the `tsconfig.node.json` file, so things like intellisense should work correctly in VSCode.

I just checked and to actually enforce this, we'd need to specify `"lib": ["esnext"],` in `tsconfig.node.json`, since the default imports the DOM libs. It adds a few new errors, so it might be out of scope for this PR though.

```suggestion
    "include": ["src/renderer/env.d.ts", "src/renderer/typed-router.d.ts", "src/renderer/**/*", "src/renderer/**/*.vue", "src/preload/*.d.ts"],
```